src/cli/templates/create/webpack.config.js

```diff
@@ -0,0 +1,149 @@
+const path = require('path')
+const minimumNodeVersion = require('minimum-node-version').default
+const nodeExternals = require('webpack-node-externals')
+const pkg = require('./package.json')
+
+const configsPromise = minimumNodeVersion().then(nodeVersion => {
+  const commonConfig = {
+    context: path.resolve(__dirname, 'src'),
+    devtool: 'source-map',
+    stats: 'minimal'
+  }
+
+  const nodeConfig = Object.assign({}, commonConfig, {
+    entry: ['./index.js'],
+    output: {
+      path: path.resolve(__dirname, 'bin'),
+      filename: 'node.bundle.js',
+      libraryTarget: 'commonjs2',
+      library: ['botpress', pkg.name]
+    },
+    resolve: {
+      extensions: ['.js'],
+      modules: [path.resolve(__dirname, 'node_modules'), 'node_modules']
+    },
+    externals: [nodeExternals(), 'botpress'],
+    target: 'node',
+    node: {
+      __filename: false,
+      __dirname: false
+    },
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
+          exclude: /node_modules/,
+          use: {
+            loader: 'babel-loader',
+            options: {
+              sourceMaps: true,
+              presets: [
+                [
+                  'env',
+                  {
+                    targets: {
+                      node: nodeVersion
+                    }
+                  }
+                ]
+              ],
+              plugins: ['transform-object-rest-spread', 'add-module-exports']
+            }
+          }
+        }
+      ]
+    }
+  })
+
+  const webConfig = Object.assign({}, commonConfig, {
+    entry: ['./views/index.jsx'],
+    output: {
+      path: path.resolve(__dirname, 'bin'),
+      publicPath: '/js/modules/',
+      filename: 'web.bundle.js',
+      libraryTarget: 'assign',
+      library: ['botpress', pkg.name]
+    },
+    resolve: {
+      extensions: ['.js', '.jsx']
+    },
+    externals: {
+      react: 'React',
+      'react-dom': 'ReactDOM'
```

Comment:
```js
'react-bootstrap': 'ReactBootstrap',
    'prop-types': 'PropTypes'
```

Missing since we migrated to React 16.

Once these are added, LGTM 👍

Reply:
Done